linux_os/guide/system/auditing/auditd_configure_rules/audit_dac_actions/audit_rules_dac_modification_umount2/rule.yml

```diff
@@ -45,6 +45,11 @@ warnings:
         have been placed independent of other system calls. Grouping these system
         calls with others as identifying earlier in this guide is more efficient.
 
+fixtext: |-
+    {{{ fixtext_audit_rules(event="umount2", key="perm_mod") | indent(4) }}}
```

Comment:
I think it would be better to have a special macro specific for the template audit_rules_dac_modification, eg. fixtext_audit_rules_dac_modification.

Reply:
What would you like to see different in that macro?

Reply:
If it would be the same text then it doesn't make sense :D